src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeResult(toFileName, getTotalData(fromFileName));
+    }
+
+    private int[] getTotalData(String fileName) {
+        File fileIn = new File(fileName);
+        int [] totalData = {0, 0};
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileIn))) {
+            String value = reader.readLine();
+
```

Comment:
```suggestion
        try (BufferedReader reader = new BufferedReader(new FileReader(fileIn))) {
            String value = reader.readLine();
```

redundant empty lines